viaq-jenkins/fields.yml

```diff
@@ -65,8 +66,10 @@ _default_:
       index: not_analyzed
       example: info
       description: >
-        Loging level as provided by: rsyslog(severitytext property), python's logging module, etc.
-        Possible values: info, warning, notice, err, debug
+        Loging level. The baseline values are provided by syslog(rfc5424): emerg, alert, crit, err, warning, notice, info, debug.
+        Any other level will be attempted to be mapped in those values by the normalizer based on semantic similarity.
```

Comment:
an attempt will be made to map any other value in to one of the base where reasonable. otherwise it'll be added as a new value.